package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "dev": "node server",
-    "build": "npm run generate-articles && npm run build:client && npm run build:server",
+    "build": "pnpm run generate-articles && npm run build:client && npm run build:server",
```

Comment:
You left a few here.

Reply:
Oh yeah, I missed them. I'll open a quick pr

Reply:
Opened: https://github.com/Vanilla-OS/website/pull/241